### ISSUE TITLE
Entity limits [ permissions ]

### ISSUE THREAD

**Issue by NickeyNexter** [BentoBoxWorld/Limits#67]
I would like that the number of entities (mobs) could be configured through permissions, like with blocks.
```
# Format is GAME-MODE-NAME.island.limit.MOB.LIMIT
# example: bskyblock.island.limit.cow.10
```

**Comment by BONNe** [comment]
https://github.com/BentoBoxWorld/Limits/blob/develop/src/main/resources/config.yml#L7

Config file, line 7... common

**Comment by BONNe** [comment]
And First PAGE in read ME...
https://github.com/BentoBoxWorld/Limits#permissions

**Comment by tastybento** [comment]
This is now available in release 1.9.2.
